src/Tooltip/Tooltip.js

```diff
@@ -355,6 +368,7 @@ class Tooltip extends React.Component {
                     className={classNames(
                       classes.tooltip,
                       { [classes.open]: open },
+                      { [classes.touch]: this.ignoreNonTouchEvents },
```

Comment:
The rule of thumb is to use a state over a class property when possible in the render method.
It's preventing edge cases, like where the render method should rerender to update but doesn't.
As long as the edge cases have been thought of and tested, I'm fine with this pattern.

Reply:
I did consider adding dedicated state, but this property already covered the requirement, so it seemed unnecessary. Because of the timeout between touchStart (when `ignoreNonTouchEvents` is set) and the Tooltip opening (state / open prop causing a rerender), I believe it's safe. A change in this property  does not itself need to trigger a render.

Happy to revisit this if it causes problems, but it appears to be working as expected.